app.py

```diff
@@ -104,6 +105,40 @@ def load_adventure_assignments_per_level(lang, level):
 
 
 app = Flask(__name__, static_url_path='')
+
+# set the security stuff https://github.com/GoogleCloudPlatform/flask-talisman
+content_security_policy = {
+    'default-src': [
+        '\'self\'',
+        '*.cloudfront.net',
+        'youtube.com',
+    ],
+    'font-src': [
+        'fonts.googleapis.com',
+        'fonts.gstatic.com',
+    ],
+    'frame-src': [
+        'www.youtube.com',
+    ],
+    'img-src': [
+        '\'self\'',
+        'data:',
```

Comment:
this seems to have mixed reviews online. If we're using inline base64 images to save on network requests, it might be just as easy to use the original png and optimize it with something like https://squoosh.app/. It technically opens up a XSS vulnerability.